code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -213,8 +213,6 @@
 			else if(is_servant_of_ratvar(M))
 				remove_servant_of_ratvar(M)
 			holder.remove_reagent(id, volume)	// maybe this is a little too perfect and a max() cap on the statuses would be better??
-			M.jitteriness = 0
-			M.stuttering = 0
```

Comment:
It'll actually take 20 seconds; the deconversion effect (and anything associated with it) should be instant and a sharp cutoff point; not a 20 seconds "yo, is it party time now"?